### DISCUSSION TITLE
검색 기능 관련 문의사항

### DISCUSSION THREAD

**Discussion by parktest0325** [min-96/_konobangumiwa#35] [Q&A]
@min-96 

현재 검색 기능이 구현되기 이전인데
왓챠와 비슷하게 키보드 입력을 받을때마다 검색 요청이 발생하는건 어떻게 생각하세요?

영어의 경우 매번 요청되게 하는건 별 문제가 없는데
한글은 ㄱ만 입력했을때 가, 감, 기 이렇게 ㄱ이 아닌 문자에 대해서도 검색이 되어야 하는데 구현이 어려운가요?

**Comment by min-96** [comment]
@parktest0325 
자바 기반에서 동작하는 검색엔진을 사용해야 될 듯 싶습니다. 
서비스 인프라환경이 달라 질 수 있으니 추후에 설치해야 될 걸 알려드리겠습니다
